client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -193,68 +212,84 @@ const BallotAudit: React.FC<IProps> = ({
 
 interface IBallotAuditContestProps {
   contest: IContest
-  interpretation: IBallotInterpretation
-  setInterpretation: (i: IBallotInterpretation) => void
+  interpretation: IBallotInterpretationFormRepresentation
+  setInterpretation: (i: IBallotInterpretationFormRepresentation) => void
 }
 
-export const INVALID_WRITE_IN = 'INVALID_WRITE_IN'
-
 const BallotAuditContest = ({
   contest,
   interpretation,
   setInterpretation,
 }: IBallotAuditContestProps) => {
+  const {
+    choiceIds,
+    comment,
+    isBlankVoteChecked,
+    isContestNotOnBallotChecked,
+    isInvalidWriteInChecked,
+  } = interpretation
+
   const onCheckboxClick = (value: string) => (
     e: React.ChangeEvent<HTMLInputElement>
   ) => {
     const { checked } = e.currentTarget
-    if (
-      value === Interpretation.BLANK ||
-      value === Interpretation.CANT_AGREE ||
-      value === Interpretation.CONTEST_NOT_ON_BALLOT
-    ) {
-      setInterpretation({
-        ...interpretation,
-        interpretation: checked ? value : null,
-        choiceIds: [],
-        hasInvalidWriteIn: false,
-      })
-    } else if (value === INVALID_WRITE_IN) {
-      let newInterpretation: Interpretation | null
-      if (interpretation.choiceIds.length > 0) {
-        newInterpretation = Interpretation.VOTE
-      } else if (checked) {
-        newInterpretation = Interpretation.BLANK
+    if (value === Interpretation.BLANK) {
+      if (checked) {
+        setInterpretation({
+          ...interpretation,
+          choiceIds: [],
+          isBlankVoteChecked: true,
+          isContestNotOnBallotChecked: false,
+          isInvalidWriteInChecked: false,
+        })
       } else {
-        newInterpretation = null
+        setInterpretation({
+          ...interpretation,
+          isBlankVoteChecked: false,
+        })
```

Comment:
I think these two branches could be collapsed. Toggling Blank Vote should completely specify the state of the checkboxes.

```ts
setInterpretation({
   ...interpretation,
   choiceIds: [],
   isBlankVoteChecked: checked,
   isContestNotOnBallotChecked: false,
   isInvalidWriteInChecked: false,
})
```

Not sure if you had them separated out on purpose.

I think the same applies for Contest Not on Ballot below

Reply:
I'd initially collapsed the branches exactly as you've described but then separated them out since I felt that the separation made the logic easier to grok. But I don't feel strongly either way and can re-collapse!